src/test/java/com/synopsys/integration/alert/util/AlertFieldStatusConverter.java

```diff
@@ -0,0 +1,22 @@
+package com.synopsys.integration.alert.util;
+
+import java.util.List;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import com.synopsys.integration.alert.common.descriptor.config.field.errors.AlertFieldStatus;
+
+public class AlertFieldStatusConverter {
```

Comment:
We have DataStructureUtils which essentially does these conversions I believe. I would verify whether this class is absolutely necessary.